src/query/storages/system/src/tables_table.rs

```diff
@@ -222,31 +222,19 @@ where TablesTable<T>: HistoryAware
             .map(|v| v.engine().as_bytes().to_vec())
             .collect();
         let engines_full: Vec<Vec<u8>> = engines.clone();
-        let created_owns: Vec<String> = database_tables
+        let created_owns: Vec<i64> = database_tables
```

Comment:
`owns` seems a typo, should it be `created_ons`?

Reply:
created_on is better